manifest.json

```
@@ -0,0 +1,4 @@
+{
+  "name": "Token Balance Oracle",
+  "description": "ACL Oracle for token membership"
```

```suggestion
  "description": "ACL Oracle enforcing token ownership"
```

It's not really "membership" since we're just checking an ERC20 rather than the TokenManager.

---

Yeah good point